tests/Core/Processor/ImageProcessorTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
I need help with this file... How do I know what to import in the test, currently if I run it it will complain with the following:

``` 
There was 1 error:

1) Tests\Core\Processor\ImageProcessorTest::testSquareShrinkSuccess
Error: Call to a member function processImage() on null

/var/www/html/tests/Core/Processor/ImageProcessorTest.php:23
``` 

even if I import with `use Core\Handler\ImageHandler;`